Failing to Extract Text on Lambda

Hi, I've just deployed the new version of your code, but I'm getting errors. In particular, when I try to run the example given on the Readme:
```sh
aws lambda invoke --function-name textractor_simple --payload '{"document_uri": "https://mozilla.github.io/pdf.js/web/compressed.tracemonkey-pldi-09.pdf", "temp_uri_prefix": "s3://text-extractor/", "text_uri": "s3://text-extractor/tracemonkey.txt"}' -
```
I get a:
```sh
{
    "StatusCode": 200
}
```
And no Errors on the lambda, but when I go see the extracted text file, it has 0 bytes, and Cloudwatch says this:
```sh
[ERROR] 2017-11-09T20:32:36.918Z 1e9cea26-c58d-11e7-9503-b7e3017ab9c2 Subprocess ['/var/task/bin/pdftotext', '-layout', '-nopgbrk', '-eol', 'unix', '/tmp/tmp8xi8qzza.pdf', '/tmp/tmpim1oc76s.txt'] returned 127:
Traceback (most recent call last):
File "/var/task/utils.py", line 8, in get_subprocess_output
output = subprocess.check_output(cmdline, **kwargs)
File "/var/lang/lib/python3.6/subprocess.py", line 336, in check_output
**kwargs).stdout
File "/var/lang/lib/python3.6/subprocess.py", line 418, in run
output=stdout, stderr=stderr)
subprocess.CalledProcessError: Command '['/var/task/bin/pdftotext', '-layout', '-nopgbrk', '-eol', 'unix', '/tmp/tmp8xi8qzza.pdf', '/tmp/tmpim1oc76s.txt']' returned non-zero exit status 127.
```

And I'm a little puzzled. I believe the `pdftotext` binary should be in the `bin/` directory of the function. Maybe the libraries are having a problem? Is it working for you?

Thanks!

--- a/README.md
+++ b/README.md
@@ -91,7 +91,7 @@ For more details about how PDF OCR extraction work here, see section on [PDF OCR
 
 #### Example
 
-    aws lambda invoke --function-name textractor_ocr --payload '{"document_uri": "https://mozilla.github.io/pdf.js/web/compressed.tracemonkey-pldi-09.pdf", "temp_uri_prefix": "s3://docbot-hippocrates-assets/", "text_uri": "s3://docbot-hippocrates-assets/tracemonkey.txt", "searchable_pdf_uri": "s3://docbot-hippocrates-assets/tracemonkey.searchable.pdf"}' -
+    aws lambda invoke --function-name textractor_ocr --payload '{"document_uri": "https://mozilla.github.io/pdf.js/web/compressed.tracemonkey-pldi-09.pdf", "temp_uri_prefix": "s3://bucket/", "text_uri": "s3://bucket/tracemonkey.txt", "searchable_pdf_uri": "s3://bucket/tracemonkey.searchable.pdf"}' -
 
     aws s3 cp s3://bucket/tracemonkey-5.txt -
 

--- a/functions/ocr/lib/pdftotext
+++ b/functions/ocr/lib/pdftotext
@@ -1,0 +1,1 @@
+../../../lib-linux_x64/pdftotext

--- a/functions/ocr/lib/tesseract
+++ b/functions/ocr/lib/tesseract
@@ -1,0 +1,1 @@
+../../../lib-linux_x64/tesseract/

--- a/functions/simple/lib/antiword
+++ b/functions/simple/lib/antiword
@@ -1,0 +1,1 @@
+../../../lib-linux_x64/antiword/

--- a/functions/simple/lib/catdoc
+++ b/functions/simple/lib/catdoc
@@ -1,0 +1,1 @@
+../../../lib-linux_x64/catdoc

--- a/functions/simple/lib/pdftotext
+++ b/functions/simple/lib/pdftotext
@@ -1,0 +1,1 @@
+../../../lib-linux_x64/pdftotext

--- a/functions/simple/lib/unrtf
+++ b/functions/simple/lib/unrtf
@@ -1,0 +1,1 @@
+../../../lib-linux_x64/unrtf

